addons/l10n_ch/data/account_fiscal_position_data.xml

```diff
@@ -2,10 +2,18 @@
 <odoo>
     <data noupdate="1">
         <!-- Fiscal Position Templates -->
+        <record id="fiscal_position_template_1" model="account.fiscal.position.template">
+            <field name="name">Schweiz National</field>
```

Comment:
@jco-odoo Could you provide me German/Italian translation for this term ?

Reply:
It is in German.  English would be: Switzerland National and Italian Svizzera nazionale